src/renderer/pages/Onboarding/Welcome/WelcomeCard.tsx

```diff
@@ -9,13 +9,15 @@ type Props = {
   iconName: IconNames;
   onClick?: () => void;
   disabled?: boolean;
+  buttonTestId?: string;
```

Comment:
Why not simple `testId`?

Reply:
WelcomeCard has many elements, I thought that it may need to have a few ids, but now only one is needed, I will change it, ty!